.github/workflows/lint-report.yaml

```
@@ -0,0 +1,14 @@
+name: Lint report
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
```

In another workflow we use v3, we should probably standardize.
```suggestion
      - uses: actions/checkout@v3
```

---

✅